src/request/providers/node.ts

```diff
@@ -473,7 +468,7 @@ export default function node(url: string, options: NodeRequestOptions = {}): Tas
 							if (contentEncodings.length) {
 								const encoding = contentEncodings.pop()!.trim();
 
-								if (encoding === '' || encoding === 'identity') {
+								if (encoding === '' || encoding.toLowerCase() === 'none' || encoding === 'identity') {
```

Comment:
Instead of lower-casing one specific case here, maybe we should lowercase the encoding as it's popped off `contentEncodings`.